config/boot_test.go

```diff
@@ -53,17 +54,53 @@ func TestLoad(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			assert.Nil(t, os.Setenv("IOC_GOLANG_CONFIG_PATH", tt.iocGolangConfigPath))
-			if err := Load(); (err != nil) != tt.wantErr {
+			if err := Load(1); (err != nil) != tt.wantErr {
 				t.Errorf("Load() error = %v, wantErr %v", err, tt.wantErr)
 			}
 		})
 	}
 }
 
+func TestLoadFixedNotFound(t *testing.T) {
+	defer clearEnv()
+
+	wd, _ := os.Getwd() // current working directory
+
+	tests := []struct {
+		name                string
+		iocGolangConfigPath string
+		wantErr             bool
+	}{
+		{
+			"test load from file",
+			filepath.Join(wd, "/test/ioc_golang.yaml"), // abs path
+			false,
+		},
+		{
+			"test load from file",
+			"/test/ioc_golang.yaml", // relative path
```

Comment:
Should this relative path be   './test/ioc-golang.yaml' ? 
@photowey

Reply:
This is a typo, when written as: "/test/ioc_golang.yaml", boot#IsNotAbsPath() returns true, so the execution result will be the same as the absolute path. At the same time, the unit test cases have been modified.